README.md

```diff
@@ -81,6 +81,12 @@ or all PRs with the word `test` in the title:
 $ repo reject [regex]
 ```
 
+### repo rename
```

Comment:
Looks like it must be moved down. The description below these added lines is for `repo reject`.